employee_dialog/lib/main.dart

```diff
@@ -1,94 +1,31 @@
+import 'package:employee/Employee profile/employee_preferences.dart';
 import 'package:flutter/material.dart';
-import 'employee.dart';
-import 'employee_detail.dart';
+import 'package:flutter/services.dart';
+import 'Employee profile/profile_page.dart';
 
-void main(){
-  runApp(const EmployeeApp());
-}
+Future main() async {
```

Comment:
I don't think you need the main to be a future, you don't use any http services here or make any requests. It's better if you change back to void main()